cordova-js-src/exec.js

```diff
@@ -259,3 +163,12 @@ execProxy.nativeCallback = function () {
 };
 
 module.exports = execProxy;
+
+if (window.webkit && window.webkit.messageHandlers && window.webkit.messageHandlers.cordova && window.webkit.messageHandlers.cordova.postMessage) {
+    // unregister the old bridge
+    cordova.define.remove('cordova/exec');
```

Comment:
We shouldn't need to do this hack if cordova/exec is already the right implementation for WKWebView